chromium_src/chrome/browser/ui/views/tabs/tab.cc

```diff
@@ -5,6 +5,16 @@
 
 #include "brave/browser/ui/views/tabs/brave_alert_indicator.h"
 
+#define BRAVE_UI_VIEWS_TABS_TAB_LAYOUT_ALERT_INDICATOR_BOUNDS \
+  bounds.set_x(std::max(contents_rect.x(),                    \
```

Comment:
can you add a comment explaining what this adjustment is and why it's needed? Same with BRAVE_UI_VIEWS_TABS_TAB_LAYOUT_TITLE below